solutions/fully-configurable/variables.tf

```diff
@@ -0,0 +1,63 @@
+##############################################################################
+# Input Variables
+##############################################################################
+
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API Key."
+  sensitive   = true
+}
+
+##############################################################################
+# S2S Authorisation Variables
+##############################################################################
+
+variable "service_map" {
+  description = "Map of unique service pairs and their authorization config."
```

Comment:
This is a complex object - consumers of the DA will not know the required format. A helper markdown doc is required that this description should link to. For example -> https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/blob/main/solutions/fully-configurable/DA-keys.md